### PR TITLE
Codegen: Mark ScalaTest tests as pending

### PR DESCRIPTION
**Pull request by ennru**
Fixes https://github.com/lightbend/kalix/issues/6600

Scala counterpart to #924 and #926

### CODE REVIEW

**Review by octonato — COMMENTED** [review]
I think it will be better to use `pending` as it express better the situation.

File: codegen/scala-gen/src/main/scala/kalix/codegen/scalasdk/impl/ActionTestKitGenerator.scala [hunk 1]
```diff
@@ -137,12 +137,12 @@ object ActionTestKitGenerator {
           |
           |  "${actionClassName}" must {
           |
-          |    "have example test that can be removed" in {
-          |      val testKit = ${actionClassName}TestKit(new $actionClassName(_))
+          |    "have example test that can be removed" pending {
```
Comment (patriknw) [inline codegen/scala-gen/src/main/scala/kalix/codegen/scalasdk/impl/ActionTestKitGenerator.scala:140]: This is not the right syntax. It should be
```
in {
  pending
```

File: codegen/scala-gen/src/main/scala/kalix/codegen/scalasdk/impl/ActionTestKitGenerator.scala [hunk 2]
```diff
@@ -137,12 +137,12 @@ object ActionTestKitGenerator {
...
+          |      val service = ${actionClassName}TestKit(new $actionClassName(_))
           |      // use the testkit to execute a command
```
Comment (patriknw) [inline codegen/scala-gen/src/main/scala/kalix/codegen/scalasdk/impl/ActionTestKitGenerator.scala:142]: Shall we have a TODO comment here to hint that this test can be implemented based on the following template and that the `pending` should be removed?

**Review by patriknw — APPROVED** [review]
LGTM

**Review by octonato — APPROVED** [review]
LGTM

**Pull request merged**